compose/src/main/java/com/bottlerocketstudios/compose/util/PreviewUtil.kt

```diff
@@ -0,0 +1,13 @@
+package com.bottlerocketstudios.compose.util
+
+import android.annotation.SuppressLint
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import com.bottlerocketstudios.compose.resources.ArchitectureDemoTheme
+
+@Composable
+fun Preview(content: @Composable () -> Unit) = ArchitectureDemoTheme(content = content)
+
+@SuppressLint("UnrememberedMutableState")
+@Composable
+fun <T> T.asState() = mutableStateOf(this)
```

Comment:
@VanyaHuamanBottleRocket @BottleRocket-Colin Should we use `asMutableState` for the name here?

Reply:
Changing now